services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -0,0 +1,1109 @@
+use std::collections::VecDeque;
+use std::ffi::{c_void, CStr, CString};
+use std::os::raw::{c_char, c_int};
+use std::path::{Path, PathBuf};
+use std::str::FromStr;
+
```

Comment:
if you avoid gaps in the use statements fmt will order and merge them all.

Reply:
Rust Analyzer seems to group imports by std, crate, and other. Do we need a convention, here? If so, can we force this?

Reply:
i guess we should let the analyzer order it

Reply:
This is how the rust analyzer imports stuff. So should I discard that and order it?

If you for personal preference dislike that, how can we setup the analyzer to do it differently?

Reply:
just leave it like that